dot_lpdev_cmds.sh

```diff
@@ -441,6 +441,8 @@ function __lpdev_node_reset {
 
 function __lpdev_node_update {
 
+  wget_args=$1
+
   URL=$(curl -s https://api.github.com/repos/livepeer/go-livepeer/releases |jq -r ".[0].assets[].browser_download_url" | grep linux)
```

Comment:
I'm not a bash expert.  What's this $1 about?

Reply:
Bash function parameters are positional, so $1 refers to the first function parameter (and $2 for the second and so forth).

In the case of the lpdev tool, there is no argument supplied to `__lpdev_node_update` so the behavior of wget is unchanged. When provisioning with Vagrant, however, we silence wget, otherwise the console outputs a bunch of nonsense due to the progress meter.

Reply:
So $wget_args silences wget in line 455?

Reply:
Yes, we pass in `__lpdev_node_update --no-verbose` and that gets passed on to wget, in the case of the vagrant provisioner. https://github.com/livepeer/devenv/pull/7/files#diff-23b6f443c01ea2efcb4f36eedfea9089R64 . For lpdev, there is no effect since there is no argument passed and $1 is empty.